.github/workflows/linters.yml

```diff
@@ -43,20 +31,7 @@ jobs:
       - name: Setup Stylelint
         run: |
           npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
-          [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css-js/.stylelintrc.json
+          [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/javascript/.stylelintrc.json
       - name: Stylelint Report
         run: npx stylelint "**/*.{css,scss}"
-eslint:
-    name: ESLint
-    runs-on: ubuntu-18.04
-    steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v1
-        with:
-          node-version: "12.x"
-      - name: Setup ESLint
-        run: |
-          npm install --save-dev eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x
-          [ -f .eslintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css-js/.eslintrc.json
-      - name: ESLint Report
-        run: npx eslint .
\ No newline at end of file
+        
```

Comment:
- [ ] Good job in implementing the changes. However, the linters file still doesn't match the linters for HTML/CSS and JavaScript projects. Just copy [this file](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.github/workflows/linters.yml) and paste it exactly as it appears in the Microverse documentation.